tests/post/features/ingress.feature

```diff
@@ -12,6 +12,13 @@ Feature: Ingress
         When we perform an HTTPS request on port 443 on a workload-plane IP
         Then the server returns 404 'Not Found'
 
+    Scenario: Create new Ingress object
```

Comment:
Done, I did not test the control-plane Ingress creation since we already create some of them as part of MetalK8s so it's already tested and this PR is not really about Control Plane Ingress (we can easily add more in the future if we want to)